python-avd/pyavd/_eos_designs/structured_config/network_services/utils_wan.py

```diff
@@ -631,20 +631,32 @@ def _filtered_internet_exit_policies_and_connections(
                     if internet_exit_policy.name in wan_interface.cv_pathfinder_internet_exit.policies
                 ]
             )
-            if not local_interfaces:
+            local_port_channel_interfaces = EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3PortChannels(
+                [
+                    wan_port_channel
+                    for wan_port_channel in self.shared_utils.wan_port_channels
+                    if internet_exit_policy.name in wan_port_channel.cv_pathfinder_internet_exit.policies
+                ]
+            )
+            if not local_interfaces and not local_port_channel_interfaces:
                 # No local interface for this policy
                 # TODO: Decide if we should raise here instead
                 continue
-
+            # fetch connections associated with given internet exit policy that
+            # applies to one or more wan interfaces (L3 interfaces, L3 Port-Channels type)
             connections = self.get_internet_exit_connections(internet_exit_policy, local_interfaces)
+            connections.extend(self.get_internet_exit_connections(internet_exit_policy, local_port_channel_interfaces))
```

Comment:
Same here, if we always want to blend the local_interfaces and local_port_channel_interfaces, we might as well put them in a single property.

Reply:
Thank you for the suggestion. 
We now extend `local_interfaces` variable to hold filtered set of L3 interfaces and L3 Port-Channel interfaces. Rest of the logic in this routine then acts on single variable - `local_interfaces`.